examples/tutorial/jupyter/execution/pandas_on_ray/cluster/README.md

```diff
@@ -0,0 +1,81 @@
+![LOGO](../../../img/MODIN_ver2_hrz.png)
+
+<center>
+<h1>Scale your pandas workflows by changing one line of code</h2>
+</center>
+
+# Exercise 5: Setting up cluster environment
+
+**GOAL**: Learn how to set up a cluster for Modin.
+
+**NOTE**: This exercise has extra requirements. Read instructions carefully before attempting. 
+
+**This exercise instructs the user on how to start a 700+ core cluster, and it is not shut down until the end of Exercise 6. Read instructions carefully.**
+
+Often in practice we have a need to exceed the capabilities of a single machine. Modin works and performs well in both local mode and in a cluster environment. The key advantage of Modin is that your notebook does not change between local development and cluster execution. Users are not required to think about how many workers exist or how to distribute and partition their data; Modin handles all of this seamlessly and transparently.
```

Comment:
```suggestion
Often in practice we have a need to exceed the capabilities of a single machine.
Modin works and performs well in both local mode and in a cluster environment.
The key advantage of Modin is that your notebook does not change between
local development and cluster execution. Users are not required to think about
how many workers exist or how to distribute and partition their data;
Modin handles all of this seamlessly and transparently.
```

Reply:
@anmyachev Unfortunately, we cannot use Modin on Ray cluster from Jupyter Notebook without using Ray Client. I also thought about running a Jupyter server on the cluster head node, but I think this is not the best idea. Do you agree with me?
You can suggest using large files, but this will take a lot of time and, I think, it will not be convenient for the user.

Reply:
> Unfortunately, we cannot use Modin on Ray cluster from Jupyter Notebook without using Ray Client.

ok

> I also thought about running a Jupyter server on the cluster head node, but I think this is not the best idea. 

@Retribution98 why?

> You can suggest using large files, but this will take a lot of time and, I think, it will not be convenient for the user.

We can add a note that this example takes a long time to complete, and indicate a place where the amount of data can be reduced (preferably a one-line change) by user.

Reply:
Deploying a Jupyter notebook is quite a complex task and requires a separate tutorial. It can be added later, but for now I just added a note about this feature.